platform/cluster/saltbase/salt/kubelet/kubelet-healthcheck

```diff
@@ -5,11 +5,9 @@
 
 
 KUBELET_HC_LOG_TAG="[kubelet-healthcheck]"
+KUBELET_LOG_TAG="kubelet["
```

Comment:
It's OK but parsing here is not ideal.

Reply:
kubelet log has signature as `kubelet[32617]:`, but I have no idea what does the number mean. Better suggestions?